policy container exit with ip6tables-legacy-save error

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

Please try to use English to describe your issue, or at least provide a snippet of English translation.
-->

Bug Report
---

Type: *bug report*

### What happened
Policy container of daemon pod exit with ip6tables-legacy-save error:

```
2022-05-10 08:11:59.242 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="raw"
2022-05-10 08:11:59.249 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.249 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="nat"
2022-05-10 08:11:59.249 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.249 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="filter"
2022-05-10 08:11:59.444 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.444 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="raw"
2022-05-10 08:11:59.444 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.444 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="mangle"
2022-05-10 08:11:59.451 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.451 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="nat"
2022-05-10 08:11:59.451 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.451 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="filter"
2022-05-10 08:11:59.846 [WARNING][24856] felix/table.go 814: iptables save failed error=exit status 1
2022-05-10 08:11:59.846 [WARNING][24856] felix/table.go 763: ip6tables-legacy-save command failed error=exit status 1 ipVersion=0x6 stderr="" table="mangle"
2022-05-10 08:11:59.846 [PANIC][24856] felix/table.go 769: ip6tables-legacy-save command failed after retries ipVersion=0x6 table="mangle"
panic: (*logrus.Entry) 0xc000992640

goroutine 205 [running]:
github.com/sirupsen/logrus.Entry.log(0xc00007e180, 0xc000269bc0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x7f6900000000, ...)
	/go/pkg/mod/github.com/projectcalico/logrus@v0.0.0-20180701205716-fc9bbf2f5799/entry.go:112 +0x2ff
github.com/sirupsen/logrus.(*Entry).Panic(0xc0005c3310, 0xc00091eb38, 0x1, 0x1)
	/go/pkg/mod/github.com/projectcalico/logrus@v0.0.0-20180701205716-fc9bbf2f5799/entry.go:182 +0xfa
github.com/sirupsen/logrus.(*Entry).Panicf(0xc0005c3310, 0x27b6396, 0x1f, 0xc00091ebf8, 0x1, 0x1)
	/go/pkg/mod/github.com/projectcalico/logrus@v0.0.0-20180701205716-fc9bbf2f5799/entry.go:230 +0xc5
github.com/projectcalico/felix/iptables.(*Table).getHashesAndRulesFromDataplane(0xc0008af000, 0xa3d43fb, 0x3caaf60)
	/go/src/github.com/projectcalico/felix/iptables/table.go:769 +0x4bc
github.com/projectcalico/felix/iptables.(*Table).loadDataplaneState(0xc0008af000)
	/go/src/github.com/projectcalico/felix/iptables/table.go:606 +0x1e5
github.com/projectcalico/felix/iptables.(*Table).Apply(0xc0008af000, 0xc0007d1fb0)
	/go/src/github.com/projectcalico/felix/iptables/table.go:990 +0x1025
github.com/projectcalico/felix/dataplane/linux.(*InternalDataplane).apply.func3(0xc000967358, 0xc000967360, 0xc0006e4c00, 0xc000967370, 0xc0008af000)
	/go/src/github.com/projectcalico/felix/dataplane/linux/int_dataplane.go:1818 +0x3c
created by github.com/projectcalico/felix/dataplane/linux.(*InternalDataplane).apply
	/go/src/github.com/projectcalico/felix/dataplane/linux/int_dataplane.go:1817 +0x6de
```

### What you expected to happen

### How to reproduce it (as minimally and precisely as possible)

### Anything else we need to know?

### Environment
- hybridnet version: v0.4.2
- OS (e.g. `cat /etc/os-release`): CentOS Linux 7
- Kernel (e.g. `uname -a`): Linux iZ0jlbn6dzzahicroo1vwhZ 3.10.0-957.21.3.el7.x86_64
- Kubernetes version: 1.20
- Install tools: ack-distro
- Others:

--- a/policy/policyinit.sh
+++ b/policy/policyinit.sh
@@ -36,7 +36,7 @@ if [ -n "$DATASTORE_TYPE" ]; then
     export FELIX_DATASTORETYPE="$DATASTORE_TYPE"
 fi
 
-if [ "$(cat /sys/module/ipv6/parameters/disable)" -ne "0" ]; then
+if [ "$(cat /sys/module/ipv6/parameters/disable)" -ne "0" ] || [ "$(cat /proc/sys/net/ipv6/conf/all/disable_ipv6)" -ne "0" ] ; then
     export FELIX_IPV6SUPPORT=false
 fi
 

